test/performance-charts/setup.ts

```diff
@@ -2,6 +2,19 @@ import { beforeAll } from 'vitest';
 import { generateLicense, LicenseInfo } from '@mui/x-license';
 
 beforeAll(() => {
+  if (process.env.VITEST_ENV !== 'browser') {
+    /* JSDOM does not impement getBBox, so we need to polyfill it. */
+    Object.defineProperty(window.SVGElement.prototype, 'getBBox', {
+      writable: true,
+      value: () => ({
+        x: 0,
+        y: 0,
+        width: 0,
+        height: 0,
+      }),
+    });
+  }
```

Comment:
Probably better to just check for presence on the `svgsvgelement`, then add it to the `svgelement` if not present. a bit confusing though 😆 

```suggestion
  if (!window.SVGSVGElement.prototype.getBBox) {
    /* JSDOM does not impement getBBox, so we need to polyfill it. */
    Object.defineProperty(window.SVGElement.prototype, 'getBBox', {
      writable: true,
      value: () => ({
        x: 0,
        y: 0,
        width: 0,
        height: 0,
      }),
    });
  }
```

Reply:
Good point 👍 [`getBBox` is defined on `SVGGraphicsElement`](https://developer.mozilla.org/en-US/docs/Web/API/SVGGraphicsElement), so I've used that prototype instead